main.py

```diff
@@ -0,0 +1,117 @@
+from datetime import datetime
+"""
+open all 3 files
+parse such that we have the log in a list
+count how many lines are in log, level=error count, level=info
+
+TIME FORMAT : 08:56:35.354
+
+top 5 most requested HTTP paths/endpoints
+"""
+
+class Logging:
+    def __init__(self):
+        self.log = []
+        self.date_time = []
+
+
+    def logs(self, file1, file2, file3):
+        file_1 = open(file1, 'r')
+        file_2 = open(file2, 'r')
+        file_3 = open(file3, 'r')
+
+        for line in file_1:
+            self.log.append(line)
+        for line in file_2:
+            self.log.append(line)
+        for line in file_3:
+            self.log.append(line)
+
+        file_1.close()
+        file_2.close()
+        file_3.close()
+
+    def get_entry_count(self):
+        return len(self.log)
+
+    def get_level_count(self, level):
+        count = 0
+
+        for line in self.log:
```

Comment:
This is brittle. What if somewhere in the `msg` field one of the URLs were "https://example.com/info". Seems like this would return that line regardless of the `level` field as a line that has `info` in it.